### PR TITLE
RELATED: FET-1259 remove namespaces

### PR DESCRIPTION
**Pull request by ivanmjartan**
JIRA: FET-1259

---

Supported PR commands:

| Command                                                 | Description                                                |
| ------------------------------------------------------- | ---------------------------------------------------------- |
| `ok to test`                                            | Re-run standard checks                                     |
| `extended check sonar`                                  | SonarQube tests                                            |
| `extended test - backstop`                              | BackstopJS tests                                           |
| **E2E Cypress tests commands - TIGER**                  |                                                            |
| `extended test - tiger-cypress - isolated <testName>`   | Run isolated tests running against recorded Tiger backend. |
| `extended test - tiger-cypress - record <testName>`     | Create a new recording for isolated Tiger tests.           |
| `extended test - tiger-cypress - integrated <testName>` | Run integrated tests against live backend                  |
| **E2E Cypress tests commands - BEAR**                   |                                                            |
| `extended test - cypress - isolated <testName>`         | Run isolated tests running against recorded Bear backend.  |
| `extended test - cypress - record <testName>`           | Create a new recording for isolated Bear tests.            |
| `extended test - cypress - integrated <testName>`       | Run integrated tests against live backend                  |
| **Compatibility matrix test commands - TIGER Backend**  |                                                            |
| `extended test - matrix-test <AIO_version>`             | Run integrated tests against AIO versions.                 |

`<testName>` in cypress commands is used to filter specfiles. Example, to run record with BEAR backend

-   Against `dashboard.spec.ts` and `drilling.spec.ts`, execute command `extended test - cypress - record dashboard,drilling`
-   Against all specfiles, execute command `extended test - cypress - record` or `extended test - cypress - record *`

`<AIO_version>` in commands is used to start test with multiple AIO instances - each instance in triggered by one jenkins build

-   To run with `master` and `stable`, execute command `extended test - matrix-test master,stable` or `extended test - matrix-test latest`
-   To run with specific version,ex: `2.3.0` and `2.3.1`, execute command `extended test - matrix-test 2.3.0,2.3.1`
-   In case `<AIO_version>` is empty, read versions from file `compTigerVersions.txt` of this repo

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `extended test - backstop` passes
-   [ ] `extended test - tiger-cypress - record` to record new mapping files (Tiger BE)
-   [ ] `extended test - cypress - record` to record new mapping files (Bear BE)
-   [ ] `extended test - tiger-cypress - isolated` passes
-   [ ] `extended test - cypress - isolated` passes
-   [ ] `extended test - tiger-cypress - integrated` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14369/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15919/ : FAILURE in 12m 04s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14384/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15935/ : FAILURE in 12m 57s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14390/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15939/ : FAILURE in 8m 03s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14391/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15940/ : SUCCESS in 11m 55s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14409/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15959/ : SUCCESS in 12m 03s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14413/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15966/ : FAILURE in 8m 24s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14416/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15968/ : SUCCESS in 12m 17s

File: libs/api-client-bear/src/catalogue.ts [hunk 1]
```diff
@@ -8,10 +8,21 @@ import { XhrModule } from "./xhr.js";
...
+    CatalogItem,
+    ICatalogGroup,
+    ILoadAvailableCatalogItemsParams,
+    ILoadAvailableCatalogItemsResponse,
+    ILoadCatalogGroupsParams,
+    ILoadCatalogGroupsResponse,
+    ILoadCatalogItemsParams,
+    ILoadCatalogItemsResponse,
+    ILoadDateDataSetsParams,
+    ItemDescription as BearItemDescription,
```
Comment (kandl) [inline libs/api-client-bear/src/catalogue.ts:20]: Looks like the exactly the same type as ItemDescription, we should use only one of them and remove the later one. (we should likely remove the one in `./interfaces` in `api-client-bear` as it just copies some `api-model-bear` interfaces?)

File: libs/api-client-bear/src/project.ts [hunk 2]
```diff
@@ -1,6 +1,21 @@
 // (C) 2007-2022 GoodData Corporation
 import { invariant } from "ts-invariant";
-import { GdcProject, GdcUser, GdcUserGroup, GdcAccessControl } from "@gooddata/api-model-bear";
+import {
+    IProjectId,
+    IUserProjectsResponse,
+    IUserProjectsParams,
+    ITimezone as IBearTimezone,
```
Comment (kandl) [inline libs/api-client-bear/src/project.ts:7]: Why is this rename needed? Looks suspicious :)

File: libs/api-client-bear/src/user.ts [hunk 3]
```diff
@@ -2,7 +2,18 @@
...
-import { GdcUser, GdcOrganization } from "@gooddata/api-model-bear";
+import {
+    IAccountInfo,
+    IAccountInfoResponse,
+    IAccountSetting,
+    IBootstrapResource,
+    ISeparators,
+    ISeparatorsResponse,
+    IWrappedAccountSetting,
+    IFeatureFlags as IBearFeatureFlags,
```
Comment (kandl) [inline libs/api-client-bear/src/user.ts:13]: Looks like the exactly the same type as IFeatureFlags, we should use only one of them and remove the later one.

File: libs/sdk-backend-bear/src/convertors/fromBackend/CatalogConverter.ts [hunk 4]
```diff
@@ -13,9 +13,26 @@ import {
...
+    IObjectMeta,
+    IAttributeDisplayForm,
+    IWrappedFact,
+    IWrappedAttribute,
+    IWrappedMetric,
+    IDateDataSetAttribute,
+    IDateDataSet,
+} from "@gooddata/api-model-bear";
+
+//import * as GdcCatalog from "@gooddata/api-model-bear/GdcCatalog";
```
Comment (kandl) [inline libs/sdk-backend-bear/src/convertors/fromBackend/CatalogConverter.ts:33]: Forgotten commented code.

File: libs/api-client-bear/src/user.ts (hunk 3 above)
Comment (ivanmjartan) [reply libs/api-client-bear/src/user.ts:13]: Fixed

File: libs/api-client-bear/src/project.ts (hunk 2 above)
Comment (ivanmjartan) [reply libs/api-client-bear/src/project.ts:7]: Type from api-client-bear ITimeZone is different than ITimezone from from interfaces this rename is needed.

File: libs/api-client-bear/src/catalogue.ts (hunk 1 above)
Comment (ivanmjartan) [reply libs/api-client-bear/src/catalogue.ts]: Fixed

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14420/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15970/ : FAILURE in 1m 51s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14431/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15977/ : SUCCESS in 12m 17s

**Comment by ivanmjartan** [comment]
extended test - backstop

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14447/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/1008/ : SUCCESS in 26m 14s

**Review by kandl — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3861

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2876/ : SUCCESS in 10m 57s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/15999/ : SUCCESS in 13m 05s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/435/ : SUCCESS in 9m 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/4032/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/757/ : FAILURE in 18s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/396/ : SUCCESS in 7m 56s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-compatibility-pipeline/261/ : FAILURE in 8m 23s

**Comment by ivanmjartan** [comment]
extended test - matrix-test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14457/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/1011/ : FAILURE in 7m 43s

**Comment by ivanmjartan** [comment]
extended test - matrix-test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14458/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/1012/ : FAILURE in 9m 08s

**Comment by tuan-tran-thanh** [comment]
LGTM

**Review by tuan-tran-thanh — APPROVED** [review]

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3861

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2877/ : SUCCESS in 9m 30s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/16004/ : SUCCESS in 14m 14s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/436/ : SUCCESS in 11m 08s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/4033/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/758/ : FAILURE in 23s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/397/ : SUCCESS in 7m 20s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-compatibility-pipeline/264/ : FAILURE in 8m 19s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3861

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2879/ : SUCCESS in 12m 45s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/16009/ : SUCCESS in 16m 10s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/438/ : SUCCESS in 12m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/4035/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/760/ : FAILURE in 19s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/399/ : SUCCESS in 9m 11s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-compatibility-pipeline/268/ : FAILURE in 9m 12s

**Comment by ivanmjartan** [comment]
extended test - matrix-test

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/14476/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-pull-request-dispatcher-pipeline/1018/ : SUCCESS in 8m 24s

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR3861

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/2880/ : SUCCESS in 9m 27s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/16017/ : SUCCESS in 11m 52s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-bear-isolated-zuul-pipeline/439/ : SUCCESS in 11m 50s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/4036/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul/761/ : FAILURE in 23s (non-voting)
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-cypress-tiger-isolated-zuul-pipeline/401/ : SUCCESS in 10m 52s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-compatibility-pipeline/273/ : SUCCESS in 6m 33s
